hail_scripts/reference_data/combine.py

```diff
@@ -104,19 +94,22 @@ def update_joined_ht_globals(
     )
 
 
-def join_hts(datasets, version, coverage_datasets=[], reference_genome='37'):
+def join_hts(datasets, version, reference_genome='37'):
+
     # Get a list of hail tables and combine into an outer join.
-    hts = [get_ht(dataset, reference_genome) for dataset in datasets]
+    non_coverage_hts = [get_ht(dataset, reference_genome) for dataset in datasets if 'coverage' not in dataset]
```

Comment:
Unclear if this is clearer or not. 

Another option here is to `join vs annotate` dependent on whether the key is `locus, alleles` or just `locus`.  That might be more explicit?

Reply:
this doesn't work on second glance... refactoring!